br: explain more about restore speed in test report (#13605)
pingcap/docs: br/br-pitr-guide.md

>
> - Snapshot data restore speed = Snapshot data size / (duration * the number of TiKV nodes)
> - Log data restore speed = Restored log data size / (duration * the number of TiKV nodes)
>

> > The snapshot data size refers to the logical size of all KVs in a single replica, not the actual amount of restored data. BR restores all replicas according to the number of replicas configured for the cluster. If the number of replicas grows, the actual amount of restored data grows as well.

Reviewer: ```suggestion
> The snapshot data size refers to the logical size of all KVs in a single replica, not the actual amount of restored data. BR restores all replicas according to the number of replicas configured for the cluster. The more replicas there are, the more data is actually restored.
```

> The default replica number for all clusters in the test is 3.

> > To improve the overall restore performance, you can modify the [`import.num-threads`](/tikv-configuration-file.md#import) item in the TiKV configuration file and the [`concurrency`](/br/use-br) option in the BR command.

Reviewer: ```suggestion
> To improve the overall restore performance, you can modify the [`import.num-threads`](/tikv-configuration-file.md#import) item in the TiKV configuration file and the [`concurrency`](/br/use-br-command-line-tool.md#common-options) option in the BR command.
```
